src/index.js

```diff
@@ -19,12 +19,12 @@ export default () => {
       setVisible(false);
     }
 
-    Keyboard.addEventListener('keyboardDidShow', onKeyboardDidShow);
-    Keyboard.addEventListener('keyboardDidHide', onKeyboardDidHide);
+    Keyboard.addListener('keyboardDidShow', onKeyboardDidShow);
+    Keyboard.addListener('keyboardDidHide', onKeyboardDidHide);
 
     return () => {
-      Keyboard.removeEventListener('keyboardDidShow', onKeyboardDidShow);
-      Keyboard.removeEventListener('keyboardDidHide', onKeyboardDidHide);
+      Keyboard.removeListener('keyboardDidShow');
+      Keyboard.removeListener('keyboardDidHide');
```

Comment:
I think you still need to pass the callback functions so they can be garbage collected.

```suggestion
      Keyboard.removeListener('keyboardDidShow', onKeyboardDidShow);
      Keyboard.removeListener('keyboardDidHide', onKeyboardDidHide);
```

Reply:
Ah - great catch.

Just corrected.